experimental/c-cpp/manager.py

```diff
@@ -398,6 +413,79 @@ def get_fuzzer_intrinsics(self, func: Dict[str, Any]) -> Dict[str, Any]:
     return fuzzer_intrinsics
 
 
+class FuzzerGenHeuristicTestConverter(FuzzHeuristicGeneratorBase):
+  """Heuristic that provides context around target function."""
+  language = 'c'
+  name = 'TestConverterPrompt'
+
+  def __init__(self, introspector_report: Dict[str, Any],
+               all_header_files: List[str], test_dir: str):
+    super().__init__(introspector_report, all_header_files, test_dir)
+    self.introspector_report = introspector_report
+    self.all_header_files = all_header_files
+    self.github_url = ''
+
+  def get_fuzzing_targets(self) -> List[Any]:
+    return []
+
+  def get_fuzzer_test_intrinsics(self, test_case: Test) -> Dict[str, Any]:
+    (headers_to_include, _,
+     build_command_includes) = self.get_header_intrinsics()
+
+    # Include any weird macros defined that does not have any values. This
+    # was found empirically to be valuable.
+    macros_defined_in_test = []
+    for line in test_case.test_content.split('\n'):
+      if '#define' in line and len(line.split(' ')) == 2:
```

Comment:
Nit: Maybe exclude commented lines later?
```c
// #define a b
/*
#define a b
*/
```

Reply:
I'll leave this for now -- it works well but ultimately I would want some stronger logic for macros (e.g. using IR/AST stuff). Will leave as is and monitor if it shows up as a limitation.